docs/introduction/oci-types.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: OCI Types
+nav_order: 3
+parent: Introduction
+---
+
+# Specification Types
+{:.no_toc}
+
+1. TOC
+{:toc}
+
+---
+
+## Understanding How Specifications Meet Goals
+
+The core of OCI are the specifications for container images, registries (called "distributions") 
```

Comment:
I don't think registries are called distributions. They're just one of the components involved in the act of distributing containers. I might replace this parenthetical with `(for distribution)` or something?

Reply:
Good call.